src/debug.c

```diff
@@ -1373,7 +1374,7 @@ void logRegisters(ucontext_t *uc) {
         (unsigned long) uc->uc_mcontext->__ss.__gs
     );
     logStackContent((void**)uc->uc_mcontext->__ss.__esp);
-    #else
+    #elif !defined(__ppc__)
```

Comment:
why no using `__POWERPC__`?
i see 4 macros(__ppc__ __powerpc __powerpc__ __POWERPC__) for POWERPC, do we need to deal with them all?

Reply:
`__POWERPC__` = `__ppc__ && __ppc64__`. Since no existing version of 10.6 (including developer builds) supports ppc64, we do not need it here, it will never be defined.

Reply:
(But it won’t hurt to use `__POWERPC__` here, if you prefer it, effect will be the same.)